### ISSUE TITLE
Support for Jupyter export options

### ISSUE THREAD

**Issue by JitseB** [microsoft/vscode#107075]
I'd very much like to export my Jupyter notebooks using VS Code. Currently I have to use terminal to do so as I would only like to include code output and not the input. I do this so I can compile my lab journals into the correct format (my professors don't give a hoot about my code smh).

The command I use for this is: `jupyter nbconvert notebook.ipynb --to=html --TemplateExporter.exclude_output=False --TemplateExporter.exclude_input=True`. TeX doesn't work that well for me as Markdown isn't supported, so I convert the HTML file to a PDF afterwards.

**To summarize**: Create a function for Jupyter notebooks so that it can exported with or without output and with or without code.
